Add tab example to MyST reference guide (#1890)
plone/documentation: docs/contributing/documentation/myst-reference.md

```
````

### Tabs

> A simple tab component without synchronization, using a basic tab-set.

Reviewer: ```suggestion
You can use [Sphinx Design](https://sphinx-design.readthedocs.io/en/latest/) to provide code examples or commands for multiple operating systems or software versions in a tabbed interface.

The following snippet is an example of a tab component without synchronization, using a basic tab-set, and will display as shown.
```

> ::::{tab-set}

Reviewer: Would you please change all the colons to backticks for consistency? I've done the first as an example.

Also let's use the `example` directive, which should provide both the source code and how it renders in a single block. It doesn't work with all Sphinx extensions, so if it doesn't work, we can remove it. Also remember to add a closing sequence of backticks for the outermost one I've added here.

``````suggestion
`````{example}
````{tab-set}
``````

:::{tab-item} Label1
Content 1
:::

:::{tab-item} Label2
Content 2
:::

::::

> A synchronized tab component, where tab selection is linked across multiple tab-sets using a unique sync key.

Reviewer: ```suggestion
The following snippet is an example of a tab component with synchronization, where tab selection is linked across multiple tab-sets using a unique sync key, and will display as shown.
```
